lib/msf/ui/debug.rb

```diff
@@ -145,7 +148,44 @@ def self.datastore(framework, driver)
           content
         )
       rescue StandardError => e
-        section_build_error('Failed to extract Datastore', e)
+        build_section(
+          'Module/Datastore',
+          ERROR_BLURB,
+          section_build_error('Failed to extract Datastore', e)
+        )
+      end
+
+      def self.database_stats(framework)
+        if framework.db&.active
+          output = "Total Workspaces: #{::Mdm::Workspace.count}\n"
+          output += "Total Hosts: #{::Mdm::Host.count}\n"
+          output += "Total Vulnerabilities: #{::Mdm::Vuln.count}\n"
+          output += "Total Notes: #{::Mdm::Note.count}\n"
+          output += "Total Services: #{::Mdm::Service.count}\n\n"
+
+          output += "Workspaces:\n\n"
+          ::Mdm::Workspace.all.each do |workspace|
+            output += "ID: #{workspace.id}\n"
+            output += "Hosts: #{workspace.hosts.count}\n"
+            output += "Vulnerabilities: #{workspace.vulns.count}\n"
+            output += "Notes: #{workspace.notes.count}\n"
+            output += "Services: #{workspace.services.count}\n\n"
```

Comment:
I found this a bit hard to read, I wonder if there's a different way to output this information 🤔 

```
Total Workspaces: 2
Total Hosts: 8
Total Vulnerabilities: 579
Total Notes: 16
Total Services: 48

Workspaces:

ID: 1
Hosts: 7
Vulnerabilities: 291
Notes: 14
Services: 27

ID: 2
Hosts: 1
Vulnerabilities: 288
Notes: 2
Services: 21
```

Reply:
Potentially a markdown table could work well here, thoughts? 🤔 

| id    | hosts | vulnerabilities | notes | services |
|-------|-------|-----------------|-------|----------|
| 1     | 1     | 291             | 14    | 27       |
| 2 (current)     | 23    | 123             | 14    | 40       |
| Total | 24    | 414             | 28    | 67       |

Reply:
We could also format the numbers with commas for extreme cases: 
```
>> 2495570.to_s(:delimited) 
=> "2,495,570"
```